### DISCUSSION TITLE
Online Conference - DevCon24  - September 3 to 5, 13.00 - 18.00 CEST

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#10246] [Announcements]
Hello,

we would like to invite you to DevCon24 conference, it is the third edition of the Developers Conference by developers for developers which is organised by Espressif Systems!

**The DevCon24 will occur online from September 3 to 5, 13.00 - 18.00 CEST.**

During the three-day online conference, we will provide you with nearly 30 talks created by Espressif technical experts, covering many topics, including ESP-IDF, **Arduino Core for ESP32**, Zephyr, RUST, and more. 

F**or more information, please visit [DevCon24](https://devcon.espressif.com/)**

This event is free and will be streamed on [Espressif’s YouTube channel](https://www.youtube.com/@EspressifSystems).

![Keynotes_DevCon24](https://github.com/user-attachments/assets/385241a3-5f16-4a8f-be80-df9c0d970326)

**Comment by Tom-Neverwinter** [comment]
opensource the blobs
